doc/getting-started/apl-nuc.rst

```diff
@@ -118,9 +118,9 @@ partition. Follow these steps:
 
       $ sudo ls -1 /boot/EFI/org.clearlinux
       bootloaderx64.efi
-      kernel-org.clearlinux.native.4.19.1-654
-      kernel-org.clearlinux.iot-lts2018-sos.4.19.0-19
-      kernel-org.clearlinux.iot-lts2018.4.19.0-19
+      kernel-org.clearlinux.native.4.19.8-670
+      kernel-org.clearlinux.iot-lts2018-sos.4.19.5-29
+      kernel-org.clearlinux.iot-lts2018.4.19.5-29
       kernel-org.clearlinux.pk414-sos.4.14.74-115
```

Comment:
Do we still have that `pk414` kernel installed (via the `service-os` bundle)? If not, can you remove this entry?